tests/execution.rs

```diff
@@ -2693,7 +2693,7 @@ fn test_reloc_64_64() {
         [],
         (),
         TestContextObject::new(3),
-        ProgramResult::Ok(ebpf::MM_BYTECODE_START),
+        ProgramResult::Ok(ebpf::MM_RODATA_START),
```

Comment:
The address of the entrypoint is `MM_BYTECODE_START`, isn't it?

Reply:
Jup, see answer for the other test.